onnxruntime/python/tools/symbolic_shape_infer.py

```diff
@@ -2389,6 +2389,29 @@ def parse_arguments():
         type=int,
         default=0,
     )
+    parser.add_argument(
+        "--save_as_exteranl_data",
+        help="Saving an ONNX model to external data",
+        action="store_true",
+        default=False,
+    )
+    parser.add_argument(
+        "--all_tensors_to_one_file",
+        help="Saving all the exteranl data to one file",
+        action="store_true",
+        default=False,
+    )
+    parser.add_argument(
+        "--external_data_location",
```

Comment:
when would one save the weights in a different location than the model file?

Reply:
I think one of the reasons to have this location parameter is that people don't want this new external weight files to be saved in the current directory that already has the old external weight files.

Reply:
Also, the onnx api has this 'location' parameter as well.
https://github.com/onnx/onnx/blob/main/docs/ExternalData.md#converting-and-saving-an-onnx-model-to-external-data

Reply:
save_model(location=...   applies to the model file too? if so, it wouldn't just be the external data location.

Reply:
Re: applies to the model file too?
No. The path of the model file is specified in the second arguments of the save_model().